charon/Makefile

```diff
@@ -3,7 +3,7 @@ all: build
 
 .PHONY: build
 build: format
-	cargo build --release
+	CARGO_PROFILE_RELEASE_DEBUG=true cargo build --release
```

Comment:
What does this do?

Reply:
That's a drive-by change that keeps more debuginfo in release mode, so that we get useful backtraces and flamegraphs. Otherwise a lot of info is lost and the flamegraphs are useless